infra/command/format

```diff
@@ -77,12 +77,7 @@ function check_newline() {
   #   .svg: xml type ML for vector graphic
   FILES_TO_CHECK_EOF=`echo "$FILES_TO_CHECK" | tr ' ' '\n' | egrep -v '((\.caffemodel)|(\.png)|(\.pdf)|(\.h5)|(\.pdf)|(\.tar.gz)|(\.tflite)|(\.pdf)|(\.bmp)|(\.svg))$'`
 
-  for f in ${FILES_TO_CHECK_EOF[@]}; do
-    # Check no new line at end of file
-    if diff /dev/null "$f" | tail -1 | grep '^\\ No newline' > /dev/null; then
-      echo >> "$f"
-    fi
-  done
+  echo "${FILES_TO_CHECK_EOF[@]}" | xargs -P $(nproc) -I {} bash -c "if diff /dev/null '{}' | tail -1 | grep -q '\\ No newline'; then echo >> '{}'; fi"
```

Comment:
I couldn't find the reason why the starting symbol '^' was not recognized in the modified code. However, I confirmed that it works well even without it.